README.md

```diff
@@ -17,9 +17,19 @@ add-on.
 
 ## Contributing changes
 
-**Pull Requests should use the `master` branch by default. Only make Pull Requests against other branches (e.g. `2.1` or `3.0`) if your changes only apply to that specific version of Godot.**
+If you'd like to contribute to the official documentation, whether you want to improve existing pages or write a new one, we would ask you to follow our workflow and guidelines:
 
-Though arguably less convenient to edit than a wiki, this Git repository is meant to receive pull requests to always improve the documentation, add new pages, etc. Having direct access to the source files in a revision control system is a big plus to ensure the quality of our documentation.
+1. If you want to write a new page, please start by opening an issue or join the discussion on an open issue with your proposal. This ensures that your work matches our [content guidelines](https://docs.godotengine.org/en/latest/community/contributing/content_guidelines.html) and saves both you and the reviewers' time.
+2. Before writing, be sure to check out our style guides:
+    - [Writing style guide](https://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html)
+    - [GDScript style guide](https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_styleguide.html) if you write GDScript code.
+    - [C# style guide](https://docs.godotengine.org/en/latest/tutorials/scripting/c_sharp/c_sharp_style_guide.html) if you write C# code.
+
+We also detail how to create new pages on the documentation [here](https://docs.godotengine.org/en/latest/community/contributing/documentation_guidelines.html).
+
+**Pull Requests should use the `master` branch by default. Only make Pull Requests against other branches (e.g. `3.2` or `2.1`) if your changes only apply to that version of Godot.**
+
+_Though less convenient to edit than a wiki, having direct access to the source files in a revision control system is a big plus to ensure the quality of our documentation._
```

Comment:
I'd say we can drop this justification by now, unless you think that it is still useful as answer to "why not a wiki?".
We wrote this initially because it was a very common question when we were setting this up, but nowadays I think most contributors understand that we chose a workflow and that asking us to change a whole infra is counterproductive.

Reply:
Sure, actually, I saw some people say we should use a wiki for the docs so everybody could participate twice since I started, but I don't think those persons read the README or the guidelines.